SetupBLT.cmake

```
@@ -33,6 +33,8 @@ if (NOT BLT_LOADED)
     # Show CMake info right out of the gate
     ################################
     message(STATUS "CMake Version: ${CMAKE_VERSION}")
+    message(STATUS "CMake Build Type: ${CMAKE_BUILD_TYPE}")
+    message(STATUS "CMake Configuration Types: ${CMAKE_CONFIGURATION_TYPE}")
```

```suggestion
    if(DEFINED CMAKE_BUILD_TYPE)
        message(STATUS "CMake Build Type: ${CMAKE_BUILD_TYPE}")
    endif()
    if(DEFINED CMAKE_CONFIGURATION_TYPES)
         message(STATUS "CMake Configuration Types: ${CMAKE_CONFIGURATION_TYPES}")
    endif()
```

I think these toggle between each other depending if it is a regular vs multi configuration build.  Also the latter variable is plural.

Can you try this out to make sure it looks correct in either case?